requirements/edx/base.txt

```diff
@@ -23,6 +23,7 @@ django-mako==0.1.5pre
 django-model-utils==2.3.1
 django-mptt==0.7.4
 django-oauth-plus==2.2.8
+django-oauth-toolkit==0.10.0
```

Comment:
I guess it isn't necessary until the next ticket.  I'll pull it out so as not to confuse matters.